modules/social_features/social_content_block/src/ContentBlockBase.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Drupal\social_content_block;
+
+use Drupal\Component\Plugin\PluginBase;
+
+/**
+ * Class ContentBlockBase.
+ *
+ * @package Drupal\social_content_block\Plugin\ContentBlock
+ */
```

Comment:
Any comment on what a ContentBlock is? How they should be used? What makes a ContentBlock different from the normal Drupal Blocks?

Reply:
Done